src/components/carousel/carousel-slide.js

```diff
@@ -11,12 +11,10 @@ export default {
   inject: {
     carousel: {
       from: 'carousel',
```

Comment:
@tmorehouse Shouldn't we prefix all injects with `bv*`?

Reply:
Yeah, we could do that... although we could need to change the provide a well.